staff/carlos-calvo/e-commerce-protos/scripts/cart.test.js

```diff
@@ -0,0 +1,253 @@
+suite("Cart", function(){
+    describe('.add function', function(){
+        it('Base Case I - Result Ok', function(){
```

Comment:
I would write a more explicative description. Also, you don't need a try catch for the positive cases. If there is an error, you will see it directly.